source/features/infinite-scroll.tsx

```diff
@@ -7,35 +7,28 @@ import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import onAbort from '../helpers/abort-controller.js';
 
-const loadMore = debounce(() => {
-	const button = select('[role="tabpanel"]:not([hidden]) button.ajax-pagination-btn')!;
+const loadMore = debounce((button: HTMLButtonElement) => {
 	button.click();
-	button.textContent = 'Loading…';
```

Comment:
Drop editing textContent since GitHub is already changes its textContent to `Loading more...`